app/src/dw/res/drawable/ooni_logo.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I assume this log is there because we want to say something like "powered by", right?